### ISSUE TITLE
Are the ResNet-18 pre-trained on ImageNet and fine-tuned for the 2.5D prediction?

### ISSUE THREAD

**Issue by daerduoCarey** [xingyuansun/pix3d#7]
Hi, Xingyuan and Jiajun,

For the MarrNet and Pix3d method, you used ResNet-18 for the image encoder. Is it pre-trained on ImageNet and fine-tuned for the 2.5D prediction task? What is the learning rate for the fine-tuning?

Thank you!

Bests,
Kaichun

**Comment by xingyuansun** [comment]
Hi Kaichun,

Thank you again for asking! Yes, we used ResNet-18 pre-trained on ImageNet and then fine-tuned it. As we mentioned in Appendix B in our paper ([https://arxiv.org/pdf/1804.04610.pdf](https://arxiv.org/pdf/1804.04610.pdf)), the learning rate was set to 2e-4. Thanks!

Best,
Xingyuan

**Comment by daerduoCarey** [comment]
Thanks for the reply. Did you use any learning rate multiplier for fine-tuning the ResNet part (e.g. making the learning rate for ResNet related weights smaller than the rest layers)? Or, you use 2e-4 for the entire network training?

Best,
Kaichun

**Comment by xingyuansun** [comment]
Thanks for asking. We did not use learning rate multiplier for fine-tuning the ResNet. Thanks!

Best,
Xingyuan
